frontend/src/utils/layers-utils.tsx

```diff
@@ -389,13 +400,10 @@ const useLayers = () => {
 
   // let users know if the layers selected are not possible to view together.
   useEffect(() => {
-    // TODO: Why is this the case here? maybe we should remove it;
     if (
-      // eslint-disable-next-line no-constant-condition
       selectedLayerDates.length !== 0 ||
       selectedLayersWithDateSupport.length === 0 ||
-      !selectedDate ||
-      1
```

Comment:
@ericboucher and @echaidemenos, I see that this logic was switched off in #1142 but we definitely do want it running now to handle situations that occur when two layers are selected with no overlapping dates (as is more common now with this composite layers). 

I've added the logic back and pulled up the Anticipatory Action module for Mozambique. Everything looks good to me but I'd love your thoughts. Anything I missed?

https://github.com/WFP-VAM/prism-app/pull/1142/files#diff-655c8639bf9b7a6f6c5aaa1270ad39e55d5500770724783a1eb36a94a8a174ae

Reply:
@ericboucher, friendly nudge here to make sure you see it when you next review (easy to miss in a PR this unruly)